bindata/network/multus/whereabouts-monitor.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: monitor-whereabouts
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: monitor-whereabouts
+  namespace: openshift-multus
+spec:
+  endpoints:
+  - interval: 5s
+    port: metrics
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    scheme: 'https'
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      serverName: whereabouts.openshift-multus.svc
+  jobLabel: app
+  namespaceSelector:
+    matchNames:
+    - openshift-multus
+  selector:
+    matchLabels:
+      app: whereabouts
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: whereabouts
+  namespace: openshift-multus
+  labels:
+    app: whereabouts
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: whereabouts-secret
+    prometheus.io/scrape: "true"
+    service.beta.openshift.io/serving-cert-secret-name: linuxptp-daemon-secret
```

Comment:
remove this line

Reply:
Which line did you want removed here? And does it matter whether the annotation field name uses "alpha" or "beta"?